tools/db_bench_tool.cc

```diff
@@ -3711,6 +3751,18 @@ void VerifyDBFromDB(std::string& truth_db_name) {
       }
       thread->stats.FinishedOps(db_with_cfh, db_with_cfh->db,
                                 entries_per_batch_, kWrite);
+      if (FLAGS_sine_write_rate) {
+        uint64_t now = FLAGS_env->NowMicros();
+        int64_t usecs_since_last = static_cast<int64_t>(now - thread->stats.GetSineInterval());
```

Comment:
This line (maybe other lines) is over 80 characters. Please wrap it.

Reply:
Right, I was actually checking out the clang-format using the google style set in the `.clang-format` for linting. It did do many changes beyond mine, and rewrote the whole file. Maybe it would be smart to standardise the linting conventions for contributors somewhere 🙂